src/Plot.js

```diff
@@ -634,6 +644,18 @@ export default class Plot extends Viz {
         s.height(barSize);
 
       }
+      else if (d.key === "Line" && this._confidence) {
+
+        const areaConfig = Object.assign({}, shapeConfig);
+        const key = this._discrete === "x" ? "y" : "x";
+        const scaleFunction = this._discrete === "x" ? y : x;
+        areaConfig[`${key}0`] = d => scaleFunction(this._confidence[0] ? d.lci : d[key]);
+        areaConfig[`${key}1`] = d => scaleFunction(this._confidence[1] ? d.hci : d[key]);
+
+        const area = new shapes.Area().config(areaConfig).fillOpacity(0.5).data(d.values);
```

Comment:
We should make this fillOpacity user-accessible. Can you make a `confidenceConfig` method (where this is the only pre-defined value). It would operate essentially the same as a shapeConfig, but only apply to the confidence shapes?